### DISCUSSION TITLE
Snakefile syntax highlighting

### DISCUSSION THREAD

**Discussion by AH-Merii** [helix-editor/helix#8526] [Q&A] [answered]
Hi all,

I have to use [snakemake](https://snakemake.readthedocs.io/en/stable/) for one of my projects, and would like to have syntax highlighting for my Snakefiles. How would I go about setting that up in helix?

**Comment by kirawi** [comment, marked as answer]
Using https://github.com/osthomas/tree-sitter-snakemake follow https://docs.helix-editor.com/guides/adding_languages.html
